### DISCUSSION TITLE
Changing the logo image/name on redash server

### DISCUSSION THREAD

**Discussion by Ultil** [getredash/redash#6223] [Q&A] [answered]
Hello - firstly, this is an amazing software. Well done o the team.

Is there a way to change the logo on our company redash server? Some of our team who will use the dashboard may get confused.

Let me know.

Best regards,
Gaurav

**Comment by justinclift** [comment, marked as answer]
We don't (yet) have a configuration option to override the existing logo.

But, what you *could* do is create a new Docker image with your own logo in place of the existing one.

I haven't tried that personally though, but it should work.  In theory. :grin:

**Reply by Ultil** [reply]
thank you for the reply - I will wait untill it is an easier process :)

**Reply by justinclift** [reply]
No worries at all. :smile:
